src/elements/emby-itemscontainer/emby-itemscontainer.js

```diff
@@ -72,8 +74,8 @@ import 'webcomponents';
         }
 
         const self = this;
-        import('multiSelect').then(({default: MultiSelect}) => {
-            self.multiSelect = new MultiSelect({
+        import('../../components/multiSelect/multiSelect').then((MultiSelect) => {
+            self.multiSelect = new MultiSelect.default({
```

Comment:
is the default still required here?

Reply:
yup it is, because is exported as default. So when you deconstruct the result, you need to use default. but one can assign an alias to it.

Reply:
fixed